tasks.py

```diff
@@ -22,6 +24,11 @@ def genspider(ctx, name, domain):
     test_filename = _gen_tests(name, domain)
     print('Created {0}'.format(test_filename))
 
+    if start_urls:
+        html_filenames = _gen_html(name, start_urls)
```

Comment:
Heh, what about `|`? Comma could work as well -- any commas in URLs could be required to be escaped. That said, somebody's probably gonna use an unescaped comma....